### ISSUE TITLE
dux_stats_column_values `pks` column got double-encoded somehow

### ISSUE THREAD

**Issue by cldellow** [cldellow/datasette-ui-extras#97]
I'm not sure how to repro this yet. I got it into a state where autosuggest was failing:

```
Traceback (most recent call last):
...
  File "/home/cldellow/src/project/db/venv/lib/python3.8/site-packages/datasette_ui_extras/column_stats.py", line 641, in autosuggest_column
    keys = list(pk.items())
AttributeError: 'str' object has no attribute 'items'
```

The underlying pks column looked like:

```
pks=['{"id":"wixlkq"}']
```

This is unexpected, it should be:

```
pks=[{"id":"wixlkq"}]
```

Running

```
datasette undux mydb.db
datasette dux mydb.db
```

fixed it.

**Comment by cldellow** [comment]
Seems to be related to pysqlite3-binary, or a newer version of SQLite? see #98

**Comment by cldellow** [comment]
Will fix it in #98, closing this one
